packages/fxa-settings/src/lib/glean/index.ts

```diff
@@ -158,7 +158,7 @@ export const GleanMetrics: Pick<
     // Starting with glean.js v2, accessing localStorage during
     // initialization could cause an error
     try {
-      if (config.enabled) {
+      if (config.enabled && !gleanEnabled) {
```

Comment:
Is this a related change?  I'm not sure why it is here.

Reply:
Agree, this is a stray change that is unrelated to this PR (sorry!). Was looking to resolve the `(Glean.core.Glean) Attempted to initialize Glean, but it has already been initialized. Ignoring.` alert that occurs on every load of settings. Will remove!